schema/causality_jwt.js

```diff
@@ -36,7 +36,9 @@ export default {
       if (options.role === 'OPERATOR' && me.type !== 'Admin') {
         throw new Error('Unauthorized to act as an operator');
       }
-      const registered = find(bidders, (b) => b.sale._id === options.sale_id);
+      const registered = find(bidders, (b) => {
+        return (b.sale._id === options.sale_id || b.sale.id === options.sale_id);
```

Comment:
Got it. If we merge as is, can you ticket a task for someone who knows the system better? Just want to make sure we don't forget to come back for it.